docs/07_maintainers_view.md

```diff
@@ -1,78 +1,70 @@
-# Maintainers View
+# Maintainers' Section
 
-So I want to document some of the views i have when maintaining NewPipe.
+These are some basic principles that I want maintainers to follow when maintaining NewPipe.
 
 
 ### Keep it Streamlined
-NewPipe is a Player for online videos on a smart phone, by means it is used
-for entertainment reason. This means it does not have to be some professional
+NewPipe is a media player for smartphones on the Android platform, thus it is intended to be used for entertainment. This means it does not have to be some professional
```

Comment:
Also works on tablets just fine. Just get rid of `for smartphones`.

Reply:
I was uncertain if it worked on tablets. Thanks for clarifying.